src/test/java/edu/ksu/canvas/attendance/submitter/AssignmentSubmitterUTest.java

```diff
@@ -202,12 +201,12 @@ public void setup() {
     }
 
     @Test
-    public void submitCourseAttendancesHappyPath() throws IOException, AttendanceAssignmentException {
+    public void submitCourseAttendancesHappyPath() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00112)